pkg/networkservice/connectioncontext/mtu/common.go

```diff
@@ -34,36 +34,7 @@ const (
 	jumboFrameSize = 9000
 )
 
-func setVPPL2MTU(ctx context.Context, conn *networkservice.Connection, vppConn api.Connection, isClient bool) error {
-	now := time.Now()
-	swIfIndex, ok := ifindex.Load(ctx, isClient)
-	if !ok || conn.GetContext().GetMTU() == 0 {
-		return nil
-	}
-	setMTU := &interfaces.HwInterfaceSetMtu{
-		SwIfIndex: swIfIndex,
-		Mtu:       uint16(conn.GetContext().GetMTU()),
-	}
-	_, err := interfaces.NewServiceClient(vppConn).HwInterfaceSetMtu(ctx, setMTU)
-	if err != nil {
-		err = errors.WithStack(err)
-		log.FromContext(ctx).
-			WithField("SwIfIndex", setMTU.SwIfIndex).
-			WithField("MTU", setMTU.Mtu).
-			WithField("duration", time.Since(now)).
-			WithField("error", err).
-			WithField("vppapi", "HwInterfaceSetMtu").Debug("error")
-		return err
-	}
-	log.FromContext(ctx).
-		WithField("SwIfIndex", setMTU.SwIfIndex).
-		WithField("MTU", setMTU.Mtu).
-		WithField("duration", time.Since(now)).
-		WithField("vppapi", "HwInterfaceSetMtu").Debug("completed")
-	return nil
-}
-
-func setVPPL3MTU(ctx context.Context, conn *networkservice.Connection, vppConn api.Connection, isClient bool) error {
+func setVPPMTU(ctx context.Context, conn *networkservice.Connection, vppConn api.Connection, isClient bool) error {
```

Comment:
Abandon setting the LinkMTU entirely as it no longer works remotely as expected.

This appears to be due to changes in VPP driven by changes in DPDK on MTU handling no longer following 802.3 on MTU handling.